cylc/flow/xtrigger_mgr.py

```diff
@@ -577,6 +574,9 @@ def load_xtrigger_for_restart(self, row_idx: int, row: Tuple[str, str]):
         # Tell the datastore this xtrigger is satisfied.
         self.data_store_mgr.delta_task_xtrigger(sig, True)
 
+        # Tell the datastore this xtrigger is satisfied.
+        self.data_store_mgr.delta_xtrigger(sig, True)
+
```

Comment:
This looks a bit odd: The two methods look like duplicates, but on examination

- They do different things - the comments probably need to explain this.
- `delta_task_xtrigger` has args `itask, label, sig, satisfied`.

Reply:
Ah yeah, sorry about that - it's due to a typo in my latest change, had it working before that.